build.sh

```diff
@@ -0,0 +1,64 @@
+set -x
+set -e
+
+export CONAN_COMPILER=clang
+export CC=${TOOLCHAIN}/bin/aarch64-linux-android23-clang
+export CXX=${TOOLCHAIN}/bin/aarch64-linux-android23-clang
+export CMAKE_C_COMPILER=${TOOLCHAIN}/bin/aarch64-linux-android23-clang
+export CMAKE_CXX_COMPILER=${TOOLCHAIN}/bin/aarch64-linux-android23-clang
+export CMAKE_LINKER=${TOOLCHAIN}/bin/aarch64-linux-android23-clang
+export CMAKE_AR=${TOOLCHAIN}/bin/aarch64-linux-android-ar
+
+export MAKE_ANDROID_NDK="$ANDROID_NDK_HOME"
+export CONAN_CMAKE_ANDROID_NDK="$ANDROID_NDK_HOME"
+
+export CMAKE_MAKE_PROGRAM="$ANDROID_NDK_HOME/prebuilt/linux-x86_64/bin/make"
+export CONAN_MAKE_PROGRAM="$CMAKE_MAKE_PROGRAM"
+printenv
+
+apt-get update -y -qq
+apt-get install -y -qq python3 python3-pip git
+
+pip3 install conan
+conan --version
+
+# cross compile profile
+mkdir -p ~/.conan/profiles
+echo "[settings]" >> ~/.conan/profiles/android
```

Comment:
Nit: might make it easier to parse if you used HEREDOC syntax rather than all these echoes + `>>`

Reply:
Next PR :)